packages/inspector/src/lib/babylon/decentraland/gizmos/ScaleGizmo.ts

```diff
@@ -34,6 +48,490 @@ export class ScaleGizmo implements IGizmoTransformer {
+    xyMaterial.alpha = 0.7;
+    xyMaterial.disableLighting = true;
+    xyPlane.material = xyMaterial;
+    xyPlane.parent = rootMesh;
+    xyPlane.isPickable = true;
+    
+    // XZ Plane (light green) - thin on Y axis, scales X and Z
+    const xzPlane = BABYLON.MeshBuilder.CreateBox('scalePlaneXZ', {
+      width: planeSize,
+      height: thickness,
+      depth: planeSize,
+    }, scene);
+    xzPlane.position.set(offset, 0, offset);
+    const xzMaterial = new BABYLON.StandardMaterial('xzPlaneMat', scene);
+    xzMaterial.diffuseColor = new BABYLON.Color3(0.5, 1.0, 0.5); // Light green (Blender style)
+    xzMaterial.emissiveColor = new BABYLON.Color3(0.3, 0.6, 0.3); // Slight glow
+    xzMaterial.alpha = 0.7;
+    xzMaterial.disableLighting = true;
+    xzPlane.material = xzMaterial;
+    xzPlane.parent = rootMesh;
+    xzPlane.isPickable = true;
+    
+    // YZ Plane (light red) - thin on X axis, scales Y and Z
+    const yzPlane = BABYLON.MeshBuilder.CreateBox('scalePlaneYZ', {
+      width: thickness,
+      height: planeSize,
+      depth: planeSize,
+    }, scene);
+    yzPlane.position.set(0, offset, offset);
+    const yzMaterial = new BABYLON.StandardMaterial('yzPlaneMat', scene);
+    yzMaterial.diffuseColor = new BABYLON.Color3(1.0, 0.5, 0.5); // Light red (Blender style)
+    yzMaterial.emissiveColor = new BABYLON.Color3(0.6, 0.3, 0.3); // Slight glow
+    yzMaterial.alpha = 0.7;
+    yzMaterial.disableLighting = true;
+    yzPlane.material = yzMaterial;
+    yzPlane.parent = rootMesh;
+    yzPlane.isPickable = true;
+    
+    // Store references for disposal later
+    this.planeCubeMeshes.push(xyPlane, xzPlane, yzPlane);
+    
+    // Store material references
+    this.planeMaterials.set(xyPlane, { 
+      material: xyMaterial, 
+      originalDiffuse: new BABYLON.Color3(0.5, 0.7, 1.0), 
+      originalEmissive: new BABYLON.Color3(0.3, 0.4, 0.6), 
+    });
+    this.planeMaterials.set(xzPlane, { 
+      material: xzMaterial, 
+      originalDiffuse: new BABYLON.Color3(0.5, 1.0, 0.5), 
+      originalEmissive: new BABYLON.Color3(0.3, 0.6, 0.3), 
+    });
+    this.planeMaterials.set(yzPlane, { 
+      material: yzMaterial, 
+      originalDiffuse: new BABYLON.Color3(1.0, 0.5, 0.5), 
+      originalEmissive: new BABYLON.Color3(0.6, 0.3, 0.3), 
+    });
+    
+    // Add hover behavior (yellow highlight like other gizmos)
+    this.addPlaneHoverBehavior(xyPlane);
+    this.addPlaneHoverBehavior(xzPlane);
+    this.addPlaneHoverBehavior(yzPlane);
+    
+    // Add drag behavior to the planes for 2-axis scaling
+    this.addPlaneDragBehavior(xyPlane, 'XY'); // Scales X and Y
+    this.addPlaneDragBehavior(xzPlane, 'XZ'); // Scales X and Z
+    this.addPlaneDragBehavior(yzPlane, 'YZ'); // Scales Y and Z
+    
+    this.planeCubesCreated = true;
+    console.log('[ScaleGizmo] Plane cubes created successfully');
+  }
+  
+  private addPlaneHoverBehavior(planeMesh: any): void {
+    const BABYLON = require('@babylonjs/core');
```

Comment:
same here, could we use imports at the top level?

`import { Color3, PointerEventTypes } from '@babylonjs/core'`